docs/detections/alerts-view-details.asciidoc

```diff
@@ -38,7 +38,7 @@ IMPORTANT: Before investigating alert prevalence data in Timeline, save any Time
 
 * *Insights*: Shows relationships with associated alerts to help you quickly identify patterns. Refer to <<alert-details-insights, Insights on alerts>> for more information.
 
-* *Enriched data*: Displays available threat indicator matches and threat intelligence data. This section only displays when examining alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
+* *Enriched data*: Displays risk scores for users and hosts and available threat intelligence data. Refer to <<alerts-enrich-host-user-risk-score>> and <<alerts-enrich-ti>> to learn more.
```

Comment:
```suggestion
* *Enriched data*: Displays risk scores for users and hosts, as well as threat intelligence data. Refer to <<alerts-enrich-host-user-risk-score>> and <<alerts-enrich-ti>> to learn more.
```

Reply:
Hm, I'm not sure if the suggested edit is correct. From my perspective, it indicates that threat intelligence is always offered for all alerts -- though feel free to correct me if I'm misinterpreting the edit. My understanding is that it only shows up if it's available in the alert's document. @nkhristinin am I relaying the correct information?

Reply:
Maybe combine the two, "risk scores for users and hosts, as well as available threat intel"? I think @benironside's edit is trying to smooth out the list of "x and y and z", so "as well as" works as long as it doesn't lose "available"?

Reply:
Gotcha, and this works^

Reply:
```suggestion
* *Enriched data*: Displays risk scores for users and hosts, as well as available threat intelligence. Refer to <<alerts-enrich-host-user-risk-score>> and <<alerts-enrich-ti>> to learn more.
```